ClassLibrary/ClassLibrary/Library.cs

```diff
@@ -21,5 +21,29 @@ public class Library
 
         public Reader GetReader(int IDReader) => departmentReaders.GetReader(IDReader);
 
+        public void ReaderBoroweBook(Reader reader,Book book)
```

Comment:
the method looks great! just change parameters to reader's ID and book title (and add some searching logic to the method to make it work)

Reply:
I did but It's not very logical (I add a new method): 
- If I went to RepositoryBooks and chose book, I was not looking for a specific book - I chose the book I like.
After that I went in DepartmentReader and add in my cardReader: I'm a Reader and I have book (with all arguments and realy book)
- I add new methods

Reply:
the idea is to have Library handle all users' (readers') requests. I know there are libraries where a reader can take books themselves, but also there libraries where you request a book and wait for it at the reception. Let's implement the second workflow.

with this in mind it should be clear now that a reader can't look for a book himself, he should ask a librarian (Library in the code) to bring it.

Please remove the method that accepts `Reader `and `Book`, leave only the second one.